sdk/python/lib/test/langhost/util.py

```diff
@@ -75,7 +75,7 @@ def RegisterResource(self, request, context):
         type_ = request.type
         name = request.name
         props = rpc.deserialize_properties(request.object)
-        deps = request.dependencies
+        deps = list(request.dependencies)
```

Comment:
This is more gRPC weirdness - `request.dependencies` returns some object that duck types as a list but is not actually a list, which confuses some test helpers like `assertListEqual` which fail if you don't give them an actual list.